src/Local/LocalBuild.php

```diff
@@ -180,7 +180,11 @@ public function getAppConfig($appRoot)
     public function getToolstack($appRoot, array $appConfig = array())
     {
         $toolstackChoice = false;
-        if (isset($appConfig['toolstack'])) {
+        // Forward compatibility with new toolstack config format.
+        if (isset($appConfig['type']) && isset($appConfig['build']['flavor'])) {
+            $toolstackChoice = $appConfig['type'] . ':' . $appConfig['build']['flavor'];
+        }
```

Comment:
`build.flavor` is actually optional, you can consider it to be `composer` (a new alias for `symfony`) if it is not specified.